monkestation/code/modules/events/new_space_law.dm

```diff
@@ -0,0 +1,36 @@
+/datum/round_event_control/new_space_law
+	name = "New Space Law"
+	typepath = /datum/round_event/new_space_law
+	max_occurrences = 1
+	weight = 15
+
+/datum/round_event/new_space_law
+	announceWhen = 1
+
+var/list/newlaw = list("alcohol is",
```

Comment:
This could actually be a strings file that the event takes from, no need for the full list to always be available

Reply:
Are there any examples of this outside of _globalvars that I could follow? _globalvars/lists/names.dm is the only place I could find strings files being used. As far as I can tell, all of the .txt files used in _globalvars are server side because they don't show up on github or my computer.

Reply:
Turns out I was being dumb and forgot githubs file search and string search are located in different locations.